Source/content/en/docs/tutorials/event-horizon.md

```diff
@@ -0,0 +1,496 @@
+                                return Task.FromResult(new PartitionedFilterResult(true, PartitionId.Unspecified));
+                            })))
+                .Build();
+            // Rest of your code here...
+        }
+    }
+}
+```
+{{% /tab %}}
+
+{{% tab name="TypeScript" %}}
+```typescript
+// index.ts
+import { Client } from '@dolittle/sdk';
+import { TenantId } from '@dolittle/sdk.execution';
+import { DishPrepared } from './DishPrepared';
+import { DishHandler } from './DishHandler';
+
+const client = Client
+    .forMicroservice('f39b1f61-d360-4675-b859-53c05c87c0e6')
+    .withEventTypes(eventTypes =>
+        eventTypes.register(DishPrepared))
+    .withEventHandlers(builder =>
+        builder.register(DishHandler))
+    .withFilters(filterBuilder =>
+        filterBuilder
+            .createPublicFilter('2c087657-b318-40b1-ae92-a400de44e507', fb =>
+                fb.handle((event: any, context: EventContext) => {
+                    console.log(`Filtering event ${event} to public stream`);
+                    return new PartitionedFilterResult(true, PartitionId.unspecified);
+                })
+            ))
+    .build();
+    // Rest of your code here...
+
+```
+{{% /tab %}}
+{{< /tabs >}}
+Notice that the returned PartitionedFilterResult always has true and the *unspecified* PartitionId. This means that this particular public stream includes all public events and they are put in the *unspecified* partition (which is the empty guid)
+
+### Commit the public event
+Now that we have a public stream we'd naturally want to commit public events to fill it up. Doing this is as easy as committing private (non-public) events. Let's show you an example where we commit a *DishPrepared* event as a public event.
+
+{{< tabs name="commit_public_tab" >}}
+{{% tab name="C#" %}}
+```csharp
+// Program.cs
+using Dolittle.SDK;
+using Dolittle.SDK.Tenancy;
+
+namespace Kitchen
+{
+    class Program
+    {
+        public static void Main()
+        {
+            // Where you build the client...
+
+            var preparedTaco = new DishPrepared("Bean Blaster Taco", "Mr. Taco");
+
+            client.EventStore
+                .ForTenant(TenantId.Development)
+                .CommitPublicEvent(preparedTaco, "bfe6f6e4-ada2-4344-8a3b-65a3e1fe16e9")
+                .GetAwaiter().GetResult();
+
+            // Blocks until the EventHandlers are finished, i.e. forever
+            client.Start().Wait();
+        }
+    }
+}
+```
+{{% /tab %}}
+
+{{% tab name="TypeScript" %}}
+```typescript
+// index.ts
+import { Client } from '@dolittle/sdk';
+import { TenantId } from '@dolittle/sdk.execution';
+import { DishPrepared } from './DishPrepared';
+import { DishHandler } from './DishHandler';
+
+// Where you build the client...
+
+const preparedTaco = new DishPrepared('Bean Blaster Taco', 'Mr. Taco');
+
+client.eventStore
+    .forTenant(TenantId.development)
+    .commit(preparedTaco, 'bfe6f6e4-ada2-4344-8a3b-65a3e1fe16e9');
```

Comment:
TypeScript missing the correct code

Reply:
https://github.com/dolittle/Documentation/pull/83/commits/4ffdbfcd742bd9e386d6f7d922c76b5a63ee7406